### PR TITLE
Create detect-av-edr-privileged-delete-vulnerability.yaml

### PR DESCRIPTION
**Pull request by BlackB0lt**
Required items, please complete
   
    Change(s):
   - Create a New rule to detect malicious files via Antivirus and EDR Elevation of Privilege Vulnerability

   Reason for Change(s):
   - New rule for detection of Privileged Delete Vulnerability in AV/EDR

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

   References: 
   - https://i.blackhat.com/EU-22/Wednesday-Briefings/EU-22-Yair-Aikido-Turning-EDRs-to-Malicious-Wipers.pdf
   - https://msrc.microsoft.com/update-guide/vulnerability/CVE-2022-37971

### CODE REVIEW

File: Hunting Queries/Microsoft 365 Defender/Privilege escalation/detect-av-edr-privileged-delete-vulnerability.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+id: a9eb9b06-4345-47f2-abe6-29f7200ddf83
+name: Windows Anitivirus and EDR Elevation of Privilege Vulnerability
+description: |
+  The query for malicious file creations via TOCTOU Vulnerability in Leading Endpoint Detection and Response (EDR) and Antivirus (AV) Solutions.
+▪ Microsoft Defender (CVE-2022-37971)
```
Comment (v-sabiraj) [inline Hunting Queries/Microsoft 365 Defender/Privilege escalation/detect-av-edr-privileged-delete-vulnerability.yaml:5]: Please remove these non ascii characters "▪", thanks.

**Comment by v-sabiraj** [comment]
@aprakash13, can you please have a look for the Hunting query added, thanks.

File: Hunting Queries/Microsoft 365 Defender/Privilege escalation/detect-av-edr-privileged-delete-vulnerability.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  - SentinelOne EDR
+  - TrendMicro Apex One (CVE-2022-45797)
+  - Avast Antivirus (CVE-2022-4173)
+  - AVG Antivirus (CVE-2022-4173)
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceFileEvents
+tactics:
+- Privilege escalation
```
Comment (v-sabiraj) [inline Hunting Queries/Microsoft 365 Defender/Privilege escalation/detect-av-edr-privileged-delete-vulnerability.yaml:16]: @BlackB0lt, can you please change tactics to "PrivilegeEscalation" and "LateralMovement", thanks.

**Comment by v-sabiraj** [comment]
@aprakash13, can you please check on the changes, thanks.

**Review by aprakash13 — APPROVED** [review]
Thanks.

**Pull request merged**